e2e/tests/company/administrator/contractor-invite-link.spec.ts

```
@@ -5,7 +5,9 @@ import { usersFactory } from "@test/factories/users";
 import { login } from "@test/helpers/auth";
 import { expect, test } from "@test/index";
 import { DocumentTemplateType } from "@/db/enums";
-import { companies, users } from "@/db/schema";
+import { companies, companyInviteLinks, users } from "@/db/schema";
+import { db } from "@test/db";
```

ESLint

[error] `@test/db` import should occur before import of `@test/factories/companies`

(import/order)